blueflood-core/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/RollupHandler.java

```diff
@@ -53,20 +60,51 @@
     protected final Histogram numHistogramPointsReturned = Metrics.histogram(RollupHandler.class, "Histogram points returned");
 
     private static final boolean ROLLUP_REPAIR = Configuration.getInstance().getBooleanProperty(CoreConfig.REPAIR_ROLLUPS_ON_READ);
+    protected final DiscoveryIO discoveryHandler = loadDiscoveryModule();
```

Comment:
We do this exact same thing for all our modular code. We should look into using a real configuration library, and into a real plugin system.